app/models/course.rb

```diff
@@ -149,6 +149,7 @@ def self.entry_requirement_options_without_nil_choice
   validate :validate_enrichment
   validate :validate_course_syncable, on: :sync
   validate :validate_qualification, on: :update
+  validate :validate_start_date, on: :update
```

Comment:
```suggestion
  validate  :validate_qualification, :validate_start_date, on: :update
```

https://apidock.com/rails/ActiveRecord/Validations/ClassMethods/validate

Reply:
Done 👍